ansible/group_vars/all

```diff
@@ -42,6 +42,7 @@ whisk:
 # briefly the parameters are:
 #
 #   runtimes_registry: optional registry (with trailing slack) where to pull docker images from for runtimes and backbox images
+#   runtimes_registry_user_images: optional, this will add runtimes_registry to all images for pulling, blackbox images included
```

Comment:
this might be a separate registry in some cases i'd imagine - would it make sense either now or later to make this a string value (which may be the same as runtime_registry).